frontend/app/services/MemberService.scala

```diff
@@ -305,7 +305,7 @@ class MemberService(identityService: IdentityService,
 
   override def getUsageCountWithinTerm(subscription: Subscription, unitOfMeasure: String): Future[Option[Int]] = {
     val features = subscription.features
-    val startDate = subscription.startDate.toDateTimeAtCurrentTime()
+    val startDate = subscription.startDate.toDateTimeAtStartOfDay(DateTimeZone.forID("America/Los_Angeles"))
```

Comment:
Ha! Thanks Tom! I was wondering if there was some weird business logic reason why we couldn't use `termStartDate` after all - like maybe it doesn't work well with 6-months-free membership offer, which thankfully we're no longer doing...